woocommerce-myparcel.php

```diff
@@ -393,9 +362,26 @@ public function initSettings()
          *
          * @return bool
          */
-        private function phpVersionMeets($version)
+        private function phpVersionMeets(string $version): bool
         {
-            return version_compare(PHP_VERSION, $version, '>=');
+            if (version_compare(PHP_VERSION, $version, '>=')) {
+                return true;
+            }
+
+            $error = __('WooCommerce MyParcel requires PHP {PHP_VERSION} or higher.', 'woocommerce-myparcel');
+            $error = str_replace('{PHP_VERSION}', self::PHP_VERSION_REQUIRED, $error);
+
+            $how_to_update = __('How to update your PHP version', 'woocommerce-myparcel');
```

Comment:
camelcase!

Reply:
Is het niet eens tijd om al die creepy externe-factoren-functies naar een trait of andere class te trekken? :/ Kan je ook de hele error bijvoorbeeld abstraheren naar `<wcmp> requires <thing> <version constraints>`

Reply:
^ hoeft niet hoor, maar heel graag (als je je geroepen voelt en tijd hebt) 😛